.github/workflows/deploy.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and Publish Storybook to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "16.19.0"
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Build Storybook
```

Comment:
Add `Storybook host URL` as an environment variable. See my PR mentioned in the issue.

Reply:
![image](https://github.com/user-attachments/assets/306163e6-8e05-434d-858f-21e6b6b87483)

Hey @Spiral-Memory, are you talking about this??

Reply:
Yes